css/app.css

```diff
@@ -61,7 +61,7 @@
 }
 
 .graphiql-container .toolbar {
-  overflow-x: auto;
+  overflow-x: visible;
```

Comment:
It's defaulting to scroll for some reason, so I think hard coding it would be the best option.